lib/uuidgen.coffee

```diff
@@ -4,6 +4,7 @@ module.exports =
 
   activate: ->
       atom.commands.add 'atom-text-editor', 'uuidgen:generate': => @generate()
+      atom.commands.add 'atom-text-editor', 'uuidgen:generatecap': => @generatecap()
```

Comment:
instead of duplicating code, the `generate` function should take a boolean parameter for lowercase or uppercase, these commands would then pass the appropriate value